### DISCUSSION TITLE
Plan for 0.22 release?

### DISCUSSION THREAD

**Discussion by JMS55** [zeux/meshoptimizer#781] [General]
I'm looking forward to using the improved attribute-based simplification that was added to the main branch.

Do you have any estimated timeframe or feature roadmap for when the next release is going to be made? No worries if not, just wondering if you did already have an idea of when you're planning the next release.

Thanks for the great work as always!

**Comment by zeux** [comment]
0.22 is tentatively planned for late October/early November.
